src/utils/OneUI/OneUI.js

```diff
@@ -83,6 +84,14 @@ class OneUI {
             head.appendChild(styleElement);
         });
     }
+
+    /**
+     * Inits ReactModal and sets global params for it.
+     * @param {string} selector - css selector for the app element
+     */
+    static modalInit(selector) {
```

Comment:
Would it make sense to make this selector an optional parameter for the `OneUI.init(...)` function? Required if `Modal` is used.  Otherwise you'll have to

```
OneUI.init(initParams);
OneUI.modalInit(appSelector);
OneUI.futureInits(...);
```

Reply:
Hm, I probably missed it, but why `modalInit` lives in OneUI namespace, not in the `Modal` component itself?
it's not good for the following reasons:

- if a developer will open `Modal` component, he/she will hardly find out that there is also some implicit logic somewhere else.
- If a project (assuming that it has tree shaking set up) doesn't use `Modal` but using `OneUI.init()` then it will have the whole unnecessary `react-modal` in it's bundle.

Can it be a static method on `Modal` component itself?

Reply:
This is something you need to run once in your app for optimising modal for screen readers. Hence I put it as part of the OneUI init. However tree shaking is a good point. Could this be solved in having OneUI build modularly (which we should do anyway?)

As for the developer not being aware of it, that's would not change, regardless on where this init function lives. Hence there is a note in Storybook about it.

Reply:
> Could this be solved in having OneUI build modularly (which we should do anyway?)

I don't see how it can be solved if we require `react-modal` in `src/utils/OneUI/OneUI.js`

If we have this function in `Modal` as `Modal.init()` then it can look like this:

```js
import React from 'react';
import { render } from 'react-dom';
import OneUI from '@textkernel/oneui';
import App from 'components/App';
import { Module } from '@textkernel/oneui';

const appContainerSelector = 'main';

Module.init(appContainerSelector);

OneUI.init()
  .then((): void => {
    render(<App />, document.getElementsByTagName(appContainerSelector)[0]);
  })
  .catch(...)



```

Reply:
done